spec/nylas/collection_spec.rb

```diff
@@ -74,6 +74,22 @@ def example_instance_hash
       expect(instance.api).to eq(api)
     end
 
+    it "retrieves a single object with `view` argument in query if clauses earlier in the chain" do
```

Comment:
Since we have an `if` statement, is it worth testing the other branch of the conditional?

Reply:
Added here https://github.com/nylas/nylas-ruby/pull/293/commits/9b6069fdadfb539130683f7d4bf0a637d0e1ae2c